getting_started.md

```
@@ -148,25 +163,26 @@ CREATE (bob)-[:FRIENDS_WITH {since: 1684108800}]->(charlie)
+
+let result = await graph.query(createQuery);
 console.log("Graph created successfully!");
 {% endcapture %}
 
 {% capture java_1 %}
-String createQuery = """
-CREATE (alice:User {id: 1, name: \"Alice\", email: \"alice@example.com\"})
-CREATE (bob:User {id: 2, name: \"Bob\", email: \"bob@example.com\"})
-CREATE (charlie:User {id: 3, name: \"Charlie\", email: \"charlie@example.com\"})
+String createQuery = 
+"CREATE (alice:User {id: 1, name: \"Alice\", email: \"alice@example.com\"})" +
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)